### DISCUSSION TITLE
on_app_command_completion usage

### DISCUSSION THREAD

**Discussion by yamthedoor** [Rapptz/discord.py#9131] [Q&A]
Hey all

I am trying to use on_app_command_completion. I just want to make sure I understand it correctly. Is this event called when an interaction from any bot is completed? I have a management bot that uses Slash commands but when I use the command the on_app_command_completion function is not called. 

I assumed this event was connected to the gateway event "Interaction Create".

I want to monitor for the running of a slash command and then log that it was run from a different bot than the slash command was run on.

**Comment by Rapptz** [comment]
You can't detect whether another bot has made an interaction. The closest you can get is inspecting `Message.interaction` but it's very brittle for this purpose.
